deploying/deploying.md

```diff
@@ -45,7 +45,7 @@ Deployment is a fancy term for getting your website on the web. After building o
 
 Now we'll work on deploying your app to Heroku.
 
-We'll be combining your frontend (create-react-app) with your backend (express) and deploying it to Heroku. Your frontend contains "static" JavaScript files -- when you deploy to Heroku, Heroku turns your whole React app into a couple of static files that it will serve to the browser. No matter what data you have in the database, these files will always be the same.
+We'll be combining your front-end (create-react-app) with your back-end (express) and deploying it to Heroku. Your front-end contains "static" JavaScript files -- when you deploy to Heroku, it turns your whole React app into a couple of static files that it will serve to the browser. No matter what data you have in the database, these files will always be the same.
 
 Your backend, on the other hand, is dynamic -- when you make an API request, the backend runs javascript code to do things like reading and writing to a database. Unlike the React app, which always serves the same files to the browser, the backend will serve different information to the browser depending on what's in the database. We're going to combine your dynamic code (express), with your static code (create-react-app).
```

Comment:
After this, the first point should be **cd into your app and create 2 new directories `client` and `server`**. The reason for this is we don't now what kind of directory structure they have followed ( like React-app inside the Back-end or vice-versa). It will be more better if you add a image of the final directory structure after 2nd point.

Reply:
Hi @Aayush-1999 , it isn't part of the issue to fix this, but thanks for pointing it out.  I will make sure it gets made into a new issue.